plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -731,37 +729,23 @@ public void testWrittenStats()
 
     @Test
     @Override
-    public void testCreateTableAsSelectNegativeDate()
+    public void testVarcharCastToDateInPredicate()
     {
-        // Map date column type to varchar
-        String tableName = "negative_date_" + randomNameSuffix();
+        assertThatThrownBy(super::testVarcharCastToDateInPredicate)
+                .hasStackTraceContaining("Table partitioning must be specified using setRangePartitionColumns or addHashPartitions");
 
-        try {
-            assertUpdate(format("CREATE TABLE %s AS SELECT DATE '-0001-01-01' AS dt", tableName), 1);
-            assertQuery("SELECT * FROM " + tableName, "VALUES '-0001-01-01'");
-            assertQuery(format("SELECT * FROM %s WHERE dt = '-0001-01-01'", tableName), "VALUES '-0001-01-01'");
-        }
-        finally {
-            assertUpdate("DROP TABLE IF EXISTS " + tableName);
-        }
+        abort("TODO: implement the test for Kudu");
     }
 
     @Test
     @Override
+    @SuppressWarnings("deprecation")
     public void testDateYearOfEraPredicate()
     {
-        assertThatThrownBy(super::testDateYearOfEraPredicate)
-                .hasStackTraceContaining("Cannot apply operator: varchar = date");
-    }
-
-    @Test
-    @Override
-    public void testVarcharCastToDateInPredicate()
-    {
-        assertThatThrownBy(super::testVarcharCastToDateInPredicate)
-                .hasStackTraceContaining("Table partitioning must be specified using setRangePartitionColumns or addHashPartitions");
-
-        abort("TODO: implement the test for Kudu");
+        // Override because the connector throws an exception instead of an empty result when the value is out of supported range
+        assertQuery("SELECT orderdate FROM orders WHERE orderdate = DATE '1997-09-14'", "VALUES DATE '1997-09-14'");
+        assertThat(query("SELECT * FROM orders WHERE orderdate = DATE '-1996-09-14'"))
+                .nonTrinoExceptionFailure().hasMessageContaining("integer value out of range for Type: date column: -1448295");
```

Comment:
Could you leave a TODO comment to replace the failure with `TrinoException` in a follow-up?